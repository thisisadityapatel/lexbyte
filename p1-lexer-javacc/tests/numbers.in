0  1 9 10 123 45354 2147483647
000000 01 000502000
