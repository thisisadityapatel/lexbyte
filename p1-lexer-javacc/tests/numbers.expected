String scanned: [0]
	->Token kind: <NUMBER>
	->Token value: 0
String scanned: [1]
	->Token kind: <NUMBER>
	->Token value: 1
String scanned: [9]
	->Token kind: <NUMBER>
	->Token value: 9
String scanned: [10]
	->Token kind: <NUMBER>
	->Token value: 10
String scanned: [123]
	->Token kind: <NUMBER>
	->Token value: 123
String scanned: [45354]
	->Token kind: <NUMBER>
	->Token value: 45354
String scanned: [2147483647]
	->Token kind: <NUMBER>
	->Token value: 2147483647
String scanned: [000000]
	->Token kind: <NUMBER>
	->Token value: 0
String scanned: [01]
	->Token kind: <NUMBER>
	->Token value: 1
String scanned: [000502000]
	->Token kind: <NUMBER>
	->Token value: 502000
